### PR TITLE
Link bot commands in README.md

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

**Comment by techtonik** [comment]
Wow. It gives this info itself. :laughing:

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

File: prow/README.md [hunk 1]
```diff
@@ -177,4 +179,4 @@ traffic to reach your hook and deck deployments.
 
 Hook processes the following events: issues, issue_comment, pull_request, push, status so github web hooks should be configured to send these event types. We suggest configuring your webhooks to send everything so that future event types are covered.
 
-The content-type for all webhooks type must be application/json.  This is selectable via a dropdown in the webhook configuration page.
\ No newline at end of file
+The content-type for all webhooks type must be application/json.  This is selectable via a dropdown in the webhook configuration page.
```
Comment (krzyzacy) [inline prow/README.md]: what's the change here? a new line or something?

Comment (techtonik) [inline prow/README.md]: Yes. GitHub inserts it automatically.

Comment (krzyzacy) [inline prow/README.md]: cool, thanks :-)

**Pull request merged**